app/models/shipit/commit_deployment.rb

```diff
@@ -20,14 +21,14 @@ def create_deployment_on_github!
 
       response = begin
         create_deployment_on_github(author.github_api)
-      rescue Octokit::ClientError
-        raise if Shipit.github.api == author.github_api
-        # If the deploy author didn't gave us the permission to create the deployment we falback the the main shipit
-        # user.
-        #
-        # Octokit currently raise NotFound, but I'm convinced it should be Forbidden if the user can see the repository.
-        # So to be future proof I catch boths.
-        create_deployment_on_github(Shipit.github.api)
+                 rescue Octokit::ClientError
+                   raise if Shipit.github.api == author.github_api
+                   # If the deploy author didn't gave us the permission to create the deployment we falback the the main shipit
+                   # user.
+                   #
+                   # Octokit currently raise NotFound, but I'm convinced it should be Forbidden if the user can see the repository.
+                   # So to be future proof I catch boths.
+                   create_deployment_on_github(Shipit.github.api)
```

Comment:
This indentation is off. The `rescue` alignment rule is broken: https://github.com/rubocop-hq/rubocop/pull/7531